chicken-fries/views/about.pug

```diff
@@ -1,6 +1,17 @@
 extends layout
 
 block content
+    style(type='text/css').
+        div.redirect {
```

Comment:
This repeated style block in each view is a great example of something that should be abstracted out into global styles. Otherwise every time a change to it is made, you have to duplicate that change across multiple files.

Reply:
@markmichon I noticed that last night and fixed it, maybe I didn't push the changes to the repo. I will push it and tag you you you can review it. Thanks